xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/panels/LiveDataAdvancedPanelExtension.vue

```diff
@@ -0,0 +1,99 @@
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<!--
+  The LiveDataAdvancedPanelExtension component is used to provide
+  extensions a way to inject a custom panel.
+-->
+<template>
+  <!--
+    Uses the LivedataBaseAdvancedPanel as root element, as it handles for us
+    all the Advanced Panels default behavior
+  -->
+  <LivedataBaseAdvancedPanel
+      :class="['livedata-advanced-panel-extension', 'livedata-advanced-panel-' + extension.id]"
+      :panel-id="extension.id"
+      ref="basePanel"
+  >
+
+    <!-- Provide the panel title and icon to the `header` slot -->
+    <template #header>
+      <XWikiIcon :icon-descriptor="{name: extension.icon}"/>
+      {{ extension.title }}
```

Comment:
Do we need both ``name`` and ``title``?

Reply:
The current panels have, e.g., ` Filter... ` as name in the menu and then `Filter` as title. I wanted to give extensions the possibility to do the same.